### PR TITLE
refactor: ask the seed string interactively

### LINKED ISSUE #211: ask the seed to the user not as a option parameter
Currently we have a privacy leak but we are not on mainet so we are safe.

We should ask the user to tell us the words in a very interactive way and not with command line parameter.

### PR DESCRIPTION
**Pull request by lazyfuhrer**
Fixes #211 as discussed in discord

### CODE REVIEW

File: lampod-cli/src/args.rs [hunk 1]
```diff
@@ -111,6 +111,8 @@ pub fn parse_args() -> Result<LampoCliArgs, lexopt::Error> {
     let mut bitcoind_user: Option<String> = None;
     let mut bitcoind_pass: Option<String> = None;
     let mut mnemonic: Option<String> = None;
+
```
Comment (vincenzopalazzo) [inline lampod-cli/src/args.rs:114]:
```suggestion
```

not needed empty line

File: lampod-cli/src/args.rs [hunk 2]
```diff
@@ -159,6 +160,15 @@ pub fn parse_args() -> Result<LampoCliArgs, lexopt::Error> {
...
     }
 
+    if restore_wallet_flag {
+        println!("Enter the list of words for wallet restoration, separated by spaces:");
+        let mut var = String::new();
+        match std::io::stdin().read_line(&mut var) {
+            Ok(_) => mnemonic = Some(var.trim().to_string()),
+            Err(_) => todo!()
+        }
+    }
```
Comment (vincenzopalazzo) [inline lampod-cli/src/args.rs:170]: This is not the place where to write this code, in this file there is the argument parsing, so I suggest you to write this code inside the main.rs.

In addition, we use `radicle-term` so we should see if they support some read input field from the docs https://docs.rs/radicle-term/latest/radicle_term/

e.g: https://docs.rs/radicle-term/latest/radicle_term/textarea/fn.textarea.html

Report the screenshot on discord to decide what will be the best for this usage

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]

**Comment by vincenzopalazzo** [comment]
In addition, the lint are failing, try to run `make`

File: lampod-cli/src/args.rs [hunk 3]
```diff
@@ -150,7 +150,7 @@ pub fn parse_args() -> Result<LampoCliArgs, lexopt::Error> {
                 bitcoind_pass = Some(var);
             }
             Long("restore-wallet") => {
-                restore_wallet_flag = true;
+                mnemonic = parse_mnemonic();
```
Comment (vincenzopalazzo) [inline lampod-cli/src/args.rs:153]: Still did not change my review, this code need to be done inside the main.rs code

The arg parsing should not do any magic, keep the boolean value and handle it inside the main.rs, please

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
We are still not there, trying to clarify my self.

File: lampod-cli/src/args.rs (hunk 3 above)
Comment (lazyfuhrer) [reply lampod-cli/src/args.rs:153]: Will do it soon! Exams going on :)

**Comment by vincenzopalazzo** [comment]
@lazyfuhrer are you still working on this?

**Comment by vincenzopalazzo** [comment]
Closing this by taking it over

**Pull request closed without merging**